### ISSUE TITLE
HTML does not validate

### ISSUE THREAD

**Issue by juleskuehn** [aaact-aatia/a11y-req#39]
There are numerous issues with aria roles, nesting of elements etc. which would fail WCAG. In all there are 25 issues,
